spec.md

```diff
@@ -126,6 +127,27 @@ response if the authentication fails.
 Note: Using an authentication mechanism that is agreed to via out of band
 communications could lead to interoperability issues with other platforms.
 
+## URL Properties
+
+This specification defines the following properties that might appear within
+URLs:
+- `service_id`
+- `plan_id`
+- `instance_id`
+- `binding_id`
+- `operation`
+
+While this specification places no restriction on the set of characters used
+within these strings, it is RECOMMENDED that these properties only contain
+characters from the "Unreserved Characters" as defined by
+[RFC3986](https://www.ietf.org/rfc/rfc3986.txt#section-2.3). In other words:
```

Comment:
This link doesn't go to the section. It's the text spec, and the second link is the html spec.

Reply:
fixed